### DISCUSSION TITLE
Remediation information cannot be retrieved

### DISCUSSION THREAD

**Discussion by Nawtest** [CrowdStrike/falconpy#1124] [Ideas]
In the CrowdStrike console, under the Remediations section (remediation/combined/remediation-behaviors/v1), there is information about remediations that cannot be directly retrieved from the API. The only clear remediation information that is obtained is the spotlight part.

**Comment by jshcodes** [comment]
Hi @Nawtest!

Thank you for the request, we've relayed it. 

It may get faster attention if you also submit it to the ideas portal located here:

- [US 1](https://us-1.ideas.crowdstrike.com/)
- [US 2](https://us-2.ideas.crowdstrike.com/)
- [EU 1](https://eu-1.ideas.crowdstrike.com/)

> Select the link that matches your CrowdStrike region.
